server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -971,8 +994,32 @@ private boolean match(PartialPath fullPath, String[] prefixNodes) {
 
   }
 
-  protected MeasurementMNode getMeasurementMNode(MNode deviceMNode, String measurement) {
```

Comment:
+1, we have to keep consistent with the cluster mode for "insert into root.some.parent.series" cases.

Reply:
Thanks. But I modify getMeasurementMNode method in CMManager to be consistent with MManager, because we need TSDataType to convert MNode to MeasurementMNode.